music21/abcFormat/__init__.py

```diff
@@ -1235,11 +1235,13 @@ class ABCNote(ABCToken):
     # given a logical unit, create an object
     # may be a chord, notes, bars
 
-    def __init__(self, src=''):
+    def __init__(self, src='', carriedAccidental=None):
```

Comment:
Can you add information about what type of object a `carriedAccidental` is -- is it a string (abc accidental) or a pitch.Accidental object?

Reply:
If there is an accidental explicitly set, does this override carriedAccidental?

Reply:
New comments will clarify.